README.md

```diff
@@ -43,40 +43,39 @@ To control module's behavior, change variables' values regarding the following:
 
 ## Inputs
 
-| Name | Description | Type | Default | Required |
-|------|-------------|:----:|:-----:|:-----:|
-| allow | (Only for list constraints) List of values which should be allowed | list | `<list>` | no |
-| allow\_list\_length | The number of elements in the allow list | string | `"0"` | no |
-| constraint | The constraint to be applied | string | n/a | yes |
-| deny | (Only for list constraints) List of values which should be denied | list | `<list>` | no |
-| deny\_list\_length | The number of elements in the allow list | string | `"0"` | no |
-| enforce | If boolean constraint, whether the policy is enforced at the root; if list constraint, whether to deny all (true) or allow all | string | `""` | no |
-| exclude\_folders | List of folders to exclude from the policy | list | `<list>` | no |
-| exclude\_projects | List of projects to exclude from the policy | list | `<list>` | no |
-| folder\_id | The folder id for putting the policy | string | `""` | no |
-| organization\_id | The organization id for putting the policy | string | `""` | no |
-| policy\_type | The constraint type to work with (either 'boolean' or 'list') | string | `"list"` | no |
-| project\_id | The project id for putting the policy | string | `""` | no |
+| Name                | Description                                                                                                                    |  Type  | Default  | Required |
```

Comment:
Was this table formatted after regeneration?

Reply:
Yes, MD markers haven't been updated so doc generator didn't work :)
Fixed